pkg/webhooks/admission/pods/validate/admit_pod.go

```diff
@@ -145,6 +146,15 @@ func validatePod(pod *v1.Pod, reviewResponse *admissionv1.AdmissionResponse) str
 	return msg
 }
 
+func BelongToVcJob(pod *v1.Pod) bool {
```

Comment:
why make this public?

Reply:
I moved to utils.go, may it be used elsewhere in the future?